polkadot/node/network/statement-distribution/src/v2/mod.rs

```diff
@@ -693,15 +693,24 @@ pub(crate) async fn handle_active_leaves_update<Context>(
 			}
 		});
 
+		let maybe_claim_queue = fetch_claim_queue(ctx.sender(), new_relay_parent)
+			.await
+			.map_err(JfyiError::FetchClaimQueue)?;
+
+		let groups_per_para = determine_groups_per_para(
+			&availability_cores,
+			&group_rotation_info,
+			&maybe_claim_queue,
+			seconding_limit - 1,
```

Comment:
Nit: I would make it explicit this is actually max_candidate_depth

Reply:
done!